docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+
+**Audience**: Hyperledger Fabric network admins
+
+This guide provides overview information and details for a network admin to manage certificates (certs) in Hyperledger Fabric.
+
+### Prerequisites and Resources
+
+The following Fabric documentation resources on identities, Managed Service Providers (MSPs) and Certificate Authorities (CAs) provide context for understanding certificate management:
+
+* [Identity in Hyperledger Fabric](https://hyperledger-fabric.readthedocs.io/en/latest/identity/identity.html#identity)
+* [MSPs in Hyperledger Fabric](https://hyperledger-fabric.readthedocs.io/en/latest/membership/membership.html#membership-service-provider-msp)
+* [Registration and Enrollment](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#overview-of-registration-and-enrollment)
+* [Registering an Identity](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#register-an-identity)
+* [Enrolling an Identity](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#enroll-an-identity)
+
+
+### Key Concepts
+
+**Registration** – A username (enrollment ID) and password pair, stored in the Certificate Authority (CA). This registration is created by a CA admin user, has no expiration, and contains any required roles and attributes.
+
+**Enrollment** – An identity based on the registration username and password. The enrollment contains a public/private key pair and an X.509 certificate issued by the organization Certificate Authority (CA). The certificate encodes roles and attributes from the registration.
+
+**Identity** - A public certificate and its private key used for encryption. The public certificate is the X.509 certificate issued by the CA, while the private key is stored out of band, on a secure storage.
+
+
+### Certificate Types
+
+Hyperledger Fabric implements two types of certificates: 1) **Enrollment** certificates for identities and 2) **TLS** certificates for node and client communications.
+
+#### Enrollment Certificates
+
+Enrollment certificates are classed into four types:
+
+* **Admin**
+* **Peer**
+* **Orderer**
+* **Client**
+
+Each enrollment certificate type has a specific role:
+
+**Admin:** X.509 certificates used to authenticate admin identities, which are required to make changes to  Fabric configurations.
+
+**Peer:** X.509 certificates used to enroll peer nodes, located physically on the node or mapped to the node. For a Fabric peer node to start, it must have a valid enrollment certificate with the required attributes.
+
+**Orderer:** X.509 certificates used to enroll orderer nodes, located physically on the node or mapped to the node. For a Fabric orderer node to start, it must have a valid enrollment certificate with the required attributes.
+
+**Client:** X.509 certificates that allow signed requests to be passed from clients to Fabric nodes. These requests require a signature for admin level certificates.
```

Comment:
I'm not sure what "These requests require a signature for admin level certificates" means here. What are we trying to convey?

Reply:
idk, @jkneubuh ?